Description: Remove feature async-io, depends on deprecated tokio-io
 This also removes dep on futures
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,10 +35,6 @@
 [dependencies.byteorder]
 version = "1.0"
 
-[dependencies.futures]
-version = "0.1"
-optional = true
-
 [dependencies.ogg]
 version = "0.9"
 optional = true
@@ -47,13 +43,9 @@
 version = "1.0"
 features = ["alloc"]
 
-[dependencies.tokio-io]
-version = "0.1"
-optional = true
-
 [dev-dependencies.ogg]
 version = "0.9"
 
 [features]
-async_ogg = ["ogg", "ogg/async", "futures", "tokio-io"]
+#async_ogg = ["ogg", "ogg/async", "futures", "tokio-io"]
 default = ["ogg"]
